src/evaluation/examples/join_examples.py

```diff
@@ -1,19 +1,4 @@
-from textwrap import dedent
-
-PipelineCode = str
-
-
-# Do you have better name for this? Maybe UseCase?
-class PipelineExample:
-    name: str
-    code: PipelineCode
-    optimized_code: PipelineCode
-
-    def __init__(self, name: str, code: PipelineCode, optimized_code: PipelineCode):
-        self.name = name
-        self.code = dedent(code)
-        self.optimized_code = dedent(optimized_code)
-
+from .pipeline_example import PipelineExample
```

Comment:
don't rely on github. I moved all tests to individual files. I didn't remove tests